docs/application/dotnet/guides/nui/text.md

```diff
@@ -89,156 +97,89 @@ If no font is specified, default styles are used, and a suitable font for displa
 
 You can provide further flexibility for the various screens by mapping the logical size to a physical size in the stylesheet.
 
-<a name="align"></a>
 ### Align Text in TextLabel
 
 To align the text in a text label:
 
--   To enable text wrapping, use the `MultiLine` property of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+-   To enable text wrapping, use the `MultiLine` property of the `TextLabel` class:
 
     ```csharp
     label.MultiLine = true;
     ```
 
--   To align the text horizontally to the beginning, center, or end of the available area, set the `HorizontalAlignment` property of the `Tizen.NUI.BaseComponents.TextLabel` class with the corresponding value of the [Tizen.NUI.HorizontalAlignment](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HorizontalAlignment.html) enumeration:
+-   To align the text horizontally to the beginning, center, or end of the available area, set the `HorizontalAlignment` property of the `TextLabel` class with the corresponding value of the [HorizontalAlignment](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HorizontalAlignment.html) enumeration:
 
     ```csharp
     label.HorizontalAlignment = HorizontalAlignment.Begin;
     label.HorizontalAlignment = HorizontalAlignment.Center;
     label.HorizontalAlignment = HorizontalAlignment.End;
     ```
 
-    The following table summarizes the available values of the `Tizen.NUI.HorizontalAlignment` enumeration for both left-to-right (Latin) and right-to-left (Arabic) script. In addition, for the illustrated examples, it is assumed that the label size is greater than the minimum required size:
+    The following table summarizes the available values of the `HorizontalAlignment` enumeration for both left-to-right (Latin) and right-to-left (Arabic) script. In addition, for the illustrated examples, it is assumed that the label size is greater than the minimum required size:
 
     |  Alignment  | Left-to-right script example    |      Right-to-left script example |
     |-------------|---------------------------------|-----------------------------------|
     |  `Begin` |   ![Latin script aligned to beginning](media/LatinBegin.png) |  ![Arabic script aligned to beginning](media/ArabicBegin.png) |
     |`Center` |  ![Latin script aligned to center](media/LatinCenter.png) | ![Arabic script aligned to center](media/ArabicCenter.png) |
     | `End`   |  ![Latin script aligned to end](media/LatinEnd.png) |   ![Arabic script aligned to end](media/ArabicEnd.png) |
 
-
-
 ### UTF16 or UTF32 character encoding
 
 The text property can accept UTF16 or UTF32 character encoding:
 
-```
+```csharp
 TextLabel title = new TextLabel();
 title.Text = "\U0001f601";      //UTF-32
 title.Text += "\ud83d\ude01";   //UTF-16
 ```
 
-<a name="decorations"></a>
-### Use Decorations for TextLabel

```

Comment:
It looks like this title should be left.

\### Use Decorations for TextLabel

The following content is about text-decoration.